qubes_config/global_config.glade

```diff
@@ -1885,6 +1885,76 @@
                             <property name="top-attach">8</property>
                           </packing>
                         </child>
+                        <child>
+                          <object class="GtkBox">
+                            <property name="visible">True</property>
+                            <property name="can-focus">False</property>
+                            <child>
+                              <object class="GtkSpinButton" id="basics_preload_dispvm_threshold">
```

Comment:
I think this needs margin-start? Because in the finished window it's a couple of pixels to the left of the other controls in its column.